src/main/scala/com/bazaarvoice/sswf/model/history/HistoryFactory.scala

```diff
@@ -156,33 +157,39 @@ object HistoryFactory {
               currentStepStart = new DateTime(h.getEventTimestamp)
             }
             startedTimers.put(timerId, h)
-            None
+
+            val result = Success("Sleep started")
```

Comment:
I thought about this but in the ActivityTaskStarted event, the result is unused; in this event, the DecisionWorker will try to deserialize the StepEvent result and throw an `IllegalArgumentException`. It seemed like too broad of an API change to allow a "STARTED" value there.